src/zk.py

```diff
@@ -509,14 +513,14 @@ def acquire_lock(self, lock_type, allow_queue=False, timeout=None, read_lock=Fal
             raise ZookeeperException(f'Failed to acquire lock {lock_type}')
         logging.debug(f'Success acquire lock: {lock_type}')
 
-    def try_acquire_lock(self, lock_type=None, allow_queue=False, timeout=None, read_lock=False):
+    def try_acquire_lock(self, lock_type=None, allow_queue=False, timeout=None, read_lock=False, host_description=None):
         """
         Acquire lock (leader by default)
         """
         lock_type = lock_type or self.PRIMARY_LOCK_PATH
-        return self._acquire_lock(lock_type, allow_queue, timeout, read_lock=read_lock)
+        return self._acquire_lock(lock_type, allow_queue, timeout, read_lock=read_lock, host_description=host_description)
 
-    def release_lock(self, lock_type=None, wait=0):
+    def release_lock(self, lock_type=None, wait=0, host_description=None):
```

Comment:
Why do we need parameter `host_description` here, if we don't use it in this method?